satpy/scene.py

```diff
@@ -765,7 +771,9 @@ def _get_prereq_datasets(self, comp_id, prereq_nodes, keepables, skip=False):
             LOG.debug("Delaying generation of %s because of dependency's delayed generation: %s", comp_id, prereq_id)
             if not skip:
                 LOG.debug("Missing prerequisite for '{}': '{}'".format(comp_id, prereq_id))
```

Comment:
In this case, the prerequisite isn't missing, it is delayed. Could we modify the debug message to differentiate from where the prereq is actually missing ?